types/browserslist/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for browserslist 4.3
+// Project: https://github.com/browserslist/browserslist#readme
+// Definitions by: Dave Cardwell <https://github.com/davecardwell>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace browserslist {
+    interface Browserslist {
+        (queries?: string | ReadonlyArray<string>, opts?: Options): string[];
+
+        coverage: (browsers: ReadonlyArray<string>, stats?: Stats) => number;
+
+        clearCaches: () => void;
+    }
+
+    interface Stats {
+        [browser: string]: {
+            [version: string]: number;
+        };
+    }
+
+    interface Options {
+        path?: string;
+        env?: string;
+        stats?: Stats;
+        config?: string;
+        ignoreUnknownVersions?: boolean;
+        dangerousExtend?: boolean;
+    }
+}
+
+declare var browserslist: browserslist.Browserslist;
+
+export default browserslist;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`.

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

Reply:
Thanks for the feedback, @DanielRosenwasser. This should be fixed in 4bcadac46916c00d9463d155563e890b56a93214.